plugins/templates/templates/default.js

```diff
@@ -14,74 +14,41 @@ CKEDITOR.addTemplates( 'default', {
 		title: 'Image and Title',
 		image: 'template1.gif',
 		description: 'One main image with a title and text that surround the image.',
-		html: '<h3>' +
-			// Use src=" " so image is not filtered out by the editor as incorrect (src is required).
-			'<img src=" " alt="" style="margin-right: 10px" height="100" width="100" align="left" />' +
-			'Type the title here' +
-			'</h3>' +
-			'<p>' +
-			'Type the text here' +
-			'</p>'
+		html: getHtml('image-and-title.html')
 	},
 	{
 		title: 'Strange Template',
 		image: 'template2.gif',
 		description: 'A template that defines two columns, each one with a title, and some text.',
-		html: '<table cellspacing="0" cellpadding="0" style="width:100%" border="0">' +
-			'<tr>' +
-				'<td style="width:50%">' +
-					'<h3>Title 1</h3>' +
-				'</td>' +
-				'<td></td>' +
-				'<td style="width:50%">' +
-					'<h3>Title 2</h3>' +
-				'</td>' +
-			'</tr>' +
-			'<tr>' +
-				'<td>' +
-					'Text 1' +
-				'</td>' +
-				'<td></td>' +
-				'<td>' +
-					'Text 2' +
-				'</td>' +
-			'</tr>' +
-			'</table>' +
-			'<p>' +
-			'More text goes here.' +
-			'</p>'
+		html: getHtml('strange-template.html')
 	},
 	{
 		title: 'Text and Table',
 		image: 'template3.gif',
 		description: 'A title with some text and a table.',
-		html: '<div style="width: 80%">' +
-			'<h3>' +
-				'Title goes here' +
-			'</h3>' +
-			'<table style="width:150px;float: right" cellspacing="0" cellpadding="0" border="1">' +
-				'<caption style="border:solid 1px black">' +
-					'<strong>Table title</strong>' +
-				'</caption>' +
-				'<tr>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-				'</tr>' +
-				'<tr>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-				'</tr>' +
-				'<tr>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-					'<td>&nbsp;</td>' +
-				'</tr>' +
-			'</table>' +
-			'<p>' +
-				'Type the text here' +
-			'</p>' +
-			'</div>'
-	} ]
-} );
+		html: getHtml('text-and-table.html')
+	}
+	]
+});
+
+// Get the contents of the given html file.
+function getHtml(fileName) {
+	let path = CKEDITOR.getUrl(CKEDITOR.plugins.getPath('templates') + 'templates/html/' + fileName);
+	return fetch(path)
```

Comment:
Fetch API can't be used directly as it's not supported in Internet Explorer and the plugin should not become broken there after introducing such changes. There is [`ajax` plugin](https://ckeditor.com/cke4/addon/ajax) that should cover asynchronous requests, however it can't be loaded in here – it has to be loaded inside the plugin.

Actually the whole function that fetches HTML from external files should be moved into the plugin itself. The template can inform plugin that it stores its HTML in an external file by using e.g. `htmlFile` property instead of `html` one.